core/app/models/spree/taxon.rb

```diff
@@ -111,6 +111,10 @@ def child_index=(idx)
       move_to_child_with_index(parent, idx.to_i) unless new_record?
     end
 
+    def localized_slugs
```

Comment:
in fact, we returning here `permalinks` instead of `slugs` what method name suggesting, so, maybe we could add some alias for `permalink` or re-name this specific method ?

Reply:
That's a bit annoying, since products use `slug`... 
If I understand the terminology correctly, permalink is the whole url, whereas slug is the user-friendly part of the url. So in this case that's a slug. We could add an alias regardless, and maybe also unify all of them to use `slug` later?

Reply:
yeah, alias would fit here 👌🏻 

we could also think about move method `localized_slugs` into `TranslatableModule` to stick with DRY ? 🤔 👀 👂🏻

Reply:
But not everything translatable has slugs, right?

Reply:
right, but in case of ... things to consider

Reply:
> we could also think about move method localized_slugs into TranslatableModule to stick with DRY ? 🤔 👀 👂🏻
I think it's fine to repeat the pattern if it's just for 2 resources. If it was more than just the two, then I would consider extracting the functionality into a separate Concern e.g. `HasLocalizedSlugs`